sklearn/metrics/classification.py

```diff
@@ -1989,16 +1989,16 @@ def hamming_loss(y_true, y_pred, labels=None, sample_weight=None):
     -----
     In multiclass classification, the Hamming loss corresponds to the Hamming
     distance between ``y_true`` and ``y_pred`` which is equivalent to the
-    subset ``zero_one_loss`` function.
+    subset ``zero_one_loss`` function, when zero-one loss is normalized.
```

Comment:
I don't understand what you mean by normalized. The zero-one loss is either 0 or 1. Maybe you mean averaged over samples?

Reply:
Hi !

Please correct me if I'm wrong. I think zero-one loss is either 0 or 1 for a sample, but we are talking about a subset (so a batch) of samples here. In this case, one has two choices. One can sum the losses of the samples, or take what I would call a "normalized sum", which is the sum rescaled to be 0 if all samples are right and 1 if they are all wrong.

I use the name "normalized sum" because in sklearn.metrics.zero_one_loss, you can chose the second behaviour by setting the `normalize` parameter to True. It will be a simple sum if `normalize` is False.

To my understanding, the fact that this "normalized sum" is equal to the average over samples is a simple consequence of zero-one loss being 0 or 1 (so it is a consequence, not a property, which is why I am not so comfortable with the use of "average over samples").

The clarification I want to make in the documentation is that in multiclass classification, hamming loss is equivalent to zero-one loss if and only if zero-one loss is in its "normalized" form.

Reply:
looking simply at the diff it was not clear that you were referring to the "normalize" parameter of the function. Can you maybe just clarify that it refers to the normalize parameter. Then looking at normalize parameter it's clearer what it means. thanks